### PR TITLE
DOC: adding examples to ``any`` method of multi-dimensional array

### PR DESCRIPTION
**Pull request by MohitKumar020291**
This PR solves issue(#22266) of missing examples of `any` method in `fromnumeric.py` by adding examples. These examples have a 3D(3x2) array example, with; without axis, axis=0, axis=1 and axis=2.    
Other examples are about use of different datatypes: 
1. Integer, float and mixed 2D array.
2.  int to bool conversion of 2D array(3x3). This example will show how any method works with `0` and `1`.

### CODE REVIEW

**Comment by MohitKumar020291** [comment]
Thanks @seberg for changing the title.

File: numpy/core/fromnumeric.py [hunk 1]
```diff
@@ -2397,7 +2397,31 @@ def any(a, axis=None, out=None, keepdims=np._NoValue, *, where=np._NoValue):
...
+           [False, False],
+           [ True,  True]])
+
+    >>> np.any([[[True, False], [True, True]],
+    ...          [[False, False], [False, False]],
+    ...          [[True, True], [False, True]]], axis=2)
+    array([[ True,  True],
+           [False, False],
+           [ True,  True]])
+
```
[the remark this answers is not in the document]
Comment (MohitKumar020291) [reply numpy/core/fromnumeric.py:2424]: Thanks @rossbar for reviewing. Can I add those examples which you recommended?

**Comment by MohitKumar020291** [comment]
@rossbar changes are done. Thanks for help.

**Comment by rossbar** [comment]
> changes are done. Thanks for help.

@MohitKumar020291 it doesn't look like the docstring examples you have added have been simplified; in fact, they seem to have grown! In it's current state, I don't think the proposed additions are an improvement as they tend (IMO) to dilute the useful information. I'd be +1 on adding an example which demonstrates the use of `axis` per my suggestion above, but I'm -1 on adding the current changes.

**Comment by MohitKumar020291** [comment]
@rossbar please look at these changes.

File: numpy/core/fromnumeric.py [hunk 2]
```diff
@@ -2398,6 +2398,39 @@ def any(a, axis=None, out=None, keepdims=np._NoValue, *, where=np._NoValue):
...
+    >>> np.any(a3, axis = 0)
+    array([[ True, False,  True],
+    ...    [ True, False,  True]])
+    >>> np.any(a3, axis = 1)
+    array([[ True, False, False],
+    ...    [False, False,  True]])
+    >>> np.any(a3, axis = 2)
+    array([[ True,  True],
+    ...    [ True,  True]])
+
```
Comment (linus-md) [inline numpy/core/fromnumeric.py:2433]: I would say you should remove this. I see no added value compared to the example `a2`.

File: numpy/core/fromnumeric.py [hunk 3]
```diff
@@ -2398,6 +2398,39 @@ def any(a, axis=None, out=None, keepdims=np._NoValue, *, where=np._NoValue):
     >>> np.any([[True, False], [False, False]], where=[[False], [True]])
     False
 
+    >>> a1 = np.array([1, 0, 1], dtype=bool)
+    >>> a1
+    array([ True, False,  True])
+    >>> np.any(a1)
+    True
```
Comment (linus-md) [inline numpy/core/fromnumeric.py:2405]: Also this doesn't see, useful given the examples above.

**Review by linus-md — COMMENTED** [review]
Example 2 LGTM

**Comment by mdhaber** [comment]
Resolved merge conflicts. Couldn't push to this branch, so I opened gh-25511, retaining the original commits, so this can be closed.

**Pull request closed without merging**